release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2585,6 +2585,15 @@ As a temporary workaround, the image registry should not be configured as privat
 
 [id="ocp-telco-core-4-17-known-issues_{context}"]
 
+* When the `globallyDisableIrqLoadBalancing` field is set to `true` in the `PerformanceProfile` object, isolated CPUs are listed in the `IRQBALANCE_BANNED_CPULIST` variable instead of the `IRQBALANCE_BANNED_CPUS` variable. However, changing the value of the `globallyDisableIrqLoadBalancing` field from `true` to `false` does not update the `IRQBALANCE_BANNED_CPULIST` variable correctly. As a result, the number of CPUs available for load rebalancing does not increase, as the reserved CPUs remain in the `IRQBALANCE_BANNED_CPULIST` variable. (link:https://issues.redhat.com/browse/OCPBUGS-42323[*OCPBUGS-42323*])
```

Comment:
```suggestion
* When the `globallyDisableIrqLoadBalancing` field is set to `true` in the `PerformanceProfile` object, isolated CPUs are listed in the `IRQBALANCE_BANNED_CPULIST` variable instead of the `IRQBALANCE_BANNED_CPUS` variable. However, changing the value of the `globallyDisableIrqLoadBalancing` field from `true` to `false` does not update the `IRQBALANCE_BANNED_CPULIST` variable correctly. As a result, the number of CPUs available for load rebalancing does not increase, as the isolated CPUs remain in the `IRQBALANCE_BANNED_CPULIST` variable. (link:https://issues.redhat.com/browse/OCPBUGS-42323[*OCPBUGS-42323*])
```
Sorry about this part, I think I messed up when we were discussing this. The isolated cpus are left in the variable and only the reserved cpus remain for interrupt balancing.

Reply:
NP!
Updated as per the suggestion. @MarSik